test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -515,16 +514,12 @@ public void killAndHealNode() throws Exception {
         refreshedLayout = runtime.getLayoutView().getLayout();
         final long epochAfterHealingNode = 3L;
         // Waiting node to be healed and added back to the layout.
-        while (refreshedLayout.getEpoch() != epochAfterHealingNode) {
+        while (refreshedLayout.getEpoch() < epochAfterHealingNode) {
```

Comment:
It seems like you have these while loops in many places. I dont think this is a good idea, probably need to bound the retries.

Reply:
Done.